src/vs/workbench/contrib/void/browser/react/src/markdown/ApplyBlockHoverButtons.tsx

```diff
@@ -69,7 +69,13 @@ export const IconShell1 = ({ onClick, Icon, disabled, className, ...props }: Ico
 
 const COPY_FEEDBACK_TIMEOUT = 1500 // amount of time to say 'Copied!'
 
-export const CopyButton = ({ codeStr, toolTipName }: { codeStr: string | (() => Promise<string> | string), toolTipName: string }) => {
+interface CopyButtonProps {
+	text: string | (() => Promise<string> | string),
+	toolTipName: string,
+	className?: string
+}
+
+export const CopyButton = ({ text, toolTipName, className }: CopyButtonProps) => {
```

Comment:
This can probably be moved outside of ApplyBlockMoverButtons as a follow-up since it's being used for non-code-block text. Didn't want to make a large change on my first PR.